tests/test_filterset.py

```diff
@@ -767,8 +519,70 @@ def test_inset_char_filter(self):
         self.assertIn(p1, f)
         self.assertIn(p2, f)
 
+    def test_boolean_filter(self):
+        # Capitalized True
+        GET = {'is_active': 'True'}
+        filterset = UserFilter(GET, queryset=User.objects.all())
+        results = list(filterset)
+        self.assertEqual(len(results), 1)
+        self.assertEqual(results[0].username, 'user1')
+
+        # Lowercase True
+        GET = {'is_active': 'true'}
+        filterset = UserFilter(GET, queryset=User.objects.all())
+        results = list(filterset)
+        self.assertEqual(len(results), 1)
+        self.assertEqual(results[0].username, 'user1')
+
+        # Uppercase True
+        GET = {'is_active': 'TRUE'}
+        filterset = UserFilter(GET, queryset=User.objects.all())
+        results = list(filterset)
+        self.assertEqual(len(results), 1)
+        self.assertEqual(results[0].username, 'user1')
+
+        # Capitalized False
+        GET = {'is_active': 'False'}
+        filterset = UserFilter(GET, queryset=User.objects.all())
+        results = list(filterset)
+        self.assertEqual(len(results), 1)
+        self.assertEqual(results[0].username, 'user2')
+
+        # Lowercase False
+        GET = {'is_active': 'false'}
+        filterset = UserFilter(GET, queryset=User.objects.all())
+        results = list(filterset)
+        self.assertEqual(len(results), 1)
+        self.assertEqual(results[0].username, 'user2')
+
+        # Uppercase False
+        GET = {'is_active': 'FALSE'}
+        filterset = UserFilter(GET, queryset=User.objects.all())
+        results = list(filterset)
+        self.assertEqual(len(results), 1)
+        self.assertEqual(results[0].username, 'user2')
+
+    def test_isnull_override(self):
+        self.assertIsInstance(
+            PersonFilter().filters['best_friend__isnull'],
```

Comment:
TODO: this isnull filter is not actually an overrides, since it's created as part of the RelatedFilter